web/src/components/organisms/Project/Content/ContentDetails/hooks.ts

```diff
@@ -38,9 +48,10 @@ export default ({ itemId }: Props) => {
         Notification.error({ message: t("Failed to create item.") });
         return;
       }
+      navigate(`./${item.data.createItem.item.id}`);
```

Comment:
Is this a valid path? If it is, until now we have been writing full paths, so perhaps we should decide which way to go and do that always.

Reply:
@KaWaite , the full URL is the same except for the last part, so I thought to use a relative URL to make it short, and avoid unnecessary variables.
Changed to the full path.